File: app/src/main/java/com/fernandez/pablo/la24gnc/View/DetalleTurno/DetalleVariosFragment.java
package com.fernandez.pablo.la24gnc.View.DetalleTurno;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ListView;
import android.widget.TextView;

import com.fernandez.pablo.la24gnc.Model.LineaVenta;
import com.fernandez.pablo.la24gnc.R;
import com.fernandez.pablo.la24gnc.View.Utils.LineaVentaAdapter;

import java.util.ArrayList;

/**
 * A simple {@link Fragment} subclass.
 */
public class DetalleVariosFragment extends Fragment {

    private ListView listLineasVenta;
    private LineaVentaAdapter lineaVentaAdapter;
    private ArrayList<LineaVenta> lineasVenta;
    private TextView tvTotal;


    public DetalleVariosFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_detalle_varios, container, false);

        this.lineasVenta = new ArrayList<>();
        this.listLineasVenta = (ListView) view.findViewById(R.id.lvList);
        this.lineaVentaAdapter = new LineaVentaAdapter(lineasVenta,getActivity());
        this.listLineasVenta.setAdapter(lineaVentaAdapter);

        this.tvTotal = (TextView) view.findViewById(R.id.totalDineroVarios);

        ((DetalleTurnoActivity) getActivity()).cargarDetalleVarios();

        return view;
    }

    public void cargarListLineasVenta(ArrayList<LineaVenta> lineasVenta){
        for (LineaVenta lv:
             lineasVenta) {
            this.lineasVenta.add(lv);
        }
        this.lineaVentaAdapter.notifyDataSetChanged();
    }

    public void cargarTotalDineroVarios(String totalDineroVarios){
        this.tvTotal.setText(totalDineroVarios);
    }

}

File: app/src/main/java/com/fernandez/pablo/la24gnc/View/DetalleTurno/DetalleVariosFragment.kt
package com.fernandez.pablo.la24gnc.View.DetalleTurno


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ListView
import android.widget.TextView

import com.fernandez.pablo.la24gnc.Model.LineaVenta
import com.fernandez.pablo.la24gnc.R
import com.fernandez.pablo.la24gnc.View.Utils.LineaVentaAdapter

import java.util.ArrayList


/**
 * A simple [Fragment] subclass.
 */
class DetalleVariosFragment : Fragment() {

    private var listLineasVenta: ListView? = null
    private var lineaVentaAdapter: LineaVentaAdapter? = null
    private var lineasVenta: ArrayList<LineaVenta>? = null
    private var tvTotal: TextView? = null


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {

        // Inflate the layout for this fragment
        val view = inflater!!.inflate(R.layout.fragment_detalle_varios, container, false)

        this.lineasVenta = ArrayList()
        this.listLineasVenta = view.findViewById(R.id.lvList) as ListView
        this.lineaVentaAdapter = LineaVentaAdapter(lineasVenta!!, activity.applicationContext)
        this.listLineasVenta!!.adapter = lineaVentaAdapter

        this.tvTotal = view.findViewById(R.id.totalDineroVarios) as TextView

        (activity as DetalleTurnoActivity).cargarDetalleVarios()

        return view
    }

    fun cargarListLineasVenta(lineasVenta: ArrayList<LineaVenta>) {
        for (lv in lineasVenta) {
            this.lineasVenta!!.add(lv)
        }
        this.lineaVentaAdapter!!.notifyDataSetChanged()
    }

    fun cargarTotalDineroVarios(totalDineroVarios: String) {
        this.tvTotal!!.text = totalDineroVarios
    }

}// Required empty public constructor
